content/ru/user-manual/packs/components/element.md

```diff
@@ -37,30 +37,30 @@ The text element renders a string of text using a [font asset][4].
 | Size        | The width and height of the Element. This may be automatically calculated depending on other settings. |
 | Margin      | The distance from the edge of the element to the Anchor. This is only available when the Anchor is split (non-equal in one axis). |
 | Use Input   | If enabled, this Element is added to the list of elements that check for input and fire input related events. |
-| Layers      | The Layers to render this element into. More on Layers [here][8]. |
+| Layers (Слои)      | The Layers to render this element into. More on Layers [here][8]. |
 | Batch Group | The Batch Group that this model belongs to. More on Batching [here][9]. |
 
 ## Image Component Properties
 
-| Property | Description |
+| Property | Описание |
 |----------|-------------|
 | Rect     | Define the area of the texture asset to display. |
 | Mask     | Switch Image Element into a mask. Masks do not render into the scene, but instead limit child elements to only be rendered where this element is rendered. |
 | Texture  | The texture asset displayed. |
-| Color    | The color to tint the element. |
+| Цвет    | The color to tint the element. |
```

Comment:
here as well, property name should stay?

Reply:
yes, I will update it with the next PR